src/main/java/com/easyrest/components/owner/Details.java

```diff
@@ -0,0 +1,26 @@
+package com.easyrest.components.owner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Details {
+    private WebDriver driver;
+
+    private final By editInformationButton = By.xpath("//button [@title = 'Edit Information']");
+    private final By updateButton = By.xpath("//span [text() = 'Update']");
+    private final By cancelButton = By.xpath("//span [text() = 'Cancel']");
+
+    public Details (WebDriver driver) {this.driver = driver;}
+
+    public void clickOnButtonEditInformation() {
```

Comment:
after click we stay on the same page or go to another?

Reply:
in this page